src/libraries/System.Private.Xml/src/System/Xml/Xsl/IlGen/XmlILOptimizerVisitor.cs

```diff
@@ -114,15 +116,15 @@ protected override QilNode Visit(QilNode nd)
             }
 
             // Continue visitation
-            return base.Visit(nd);
+            return base.Visit(nd!);
```

Comment:
It's only nullable on few node types which are used as "root" nodes - I need to annotate all of them as nullable in order to do so - code is much cleaner with few `!` instead of having them all over the place